web/src/javascript/network.js

```diff
@@ -38,8 +38,11 @@ export function sendAdminRequest(url, method, data = {}) {
     body: JSON.stringify(data),
   }).then(response => response.json())
     .then(response => {
-      if(response.data !== "ok")
+      if(response.data !== "ok"){
+        if(response.error === "forbidden")
```

Comment:
I meant flaky in the "fragile" sense that it depends on a specific error message defined by us (that could change anytime for readability, for example), while the status code is a standard HTTP code.

Right, I wanted to separate them because I thought the error handling was different at first, but it's the same thanks to our `{ error: ... }` JSON structure.

Something like this (not tested) could make us depend on the status code instead of an error message:
```js
return fetch(adminApiURL + url, {
     method,
     headers: new Headers({
       "Content-Type": "application/json",
     }),
      body: JSON.stringify(data),
    }).then(response => {
      if (response.status === 403) window.location.href = "/web/adminLogin.html";
    }).then(response => response.json())
      .then(response => {
        if (!response.data) throw response.error;
        else response.data;
      });
```